geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/AbstractIngest.scala

```diff
@@ -53,13 +58,13 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
   def beforeRunTasks(): Unit
 
   /**
-   * Create a local ingestion converter
-   *
-   * @param file file being operated on
-   * @param failures used to tracks failures
-   * @return local converter
-   */
-  def createLocalConverter(file: File, failures: AtomicLong): LocalIngestConverter
+    * Create a local ingestion converter
```

Comment:
might want to move this back on the spacing.

Reply:
this is the 'new' scala-approved spacing, as compared to the old java style...

Reply:
hahahaha ok :) when did that start!

Reply:
it was a while ago - I'm just a slave to intellij :)